test/runScript.spec.js

```diff
@@ -100,10 +100,13 @@ describe('runScript', () => {
   })
 
   it('should not pass `gitDir` as `cwd` to `execa()` if a non-git binary is called', async () => {
-    const [linter] = runScript(['jest'], ['test.js'], scripts, { gitDir: '../' })
+    const processCwdBkp = process.cwd
+    process.cwd = () => __dirname
+    const [linter] = runScript(['jest'], ['test.js'], scripts, {})
     await linter.task()
     expect(mockFn).toHaveBeenCalledTimes(1)
     expect(mockFn).lastCalledWith('jest', ['test.js'], {})
+    process.cwd = processCwdBkp
```

Comment:
why we have this line after assertions?

Reply:
This test checks that when `gitDir != projectRoot`, and the `bin` being executed is not `git`, we don't pass the `cwd` option to `execa`. I am 'mocking' this behavior by modifying `process.cwd` to return the current dir, which is in the `test` dir. That way, `resolveGitDir` would return the parent directory, the actual project root. At the end of the test, I am resetting `process.cwd`. Does that answer your question?